en/canary-deployment-tidb-operator.md

```diff
@@ -0,0 +1,181 @@
+
+    ```yaml
+    controllerManager:
+      selector:
+      - user=dev
+    ```
+
+2. Deploy the TiDB cluster.
+
+    1. Refer to [Configure the TiDB Cluster](configure-a-tidb-cluster.md) to configure the TidbCluster CR, and configure `labels` to match the `selector` set in the last step. For example:
+
+        ```yaml
+        apiVersion: pingcap.com/v1alpha1
+        kind: TidbCluster
+        metadata:
+          name: basic1
+          labels:
+            user: dev
+        spec:
+          ...
+        ```
+
+        If `label` is not set when you deploy the TiDB cluster, you can configure `label` by running the following command:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        kubectl -n ${namespace} label tidbcluster ${cluster_name} user=dev
+        ```
+
+    2. Refer to [Deploy TiDB in General Kubernetes](deploy-on-general-kubernetes.md) to deploy the TiDB cluster. Confirm that each component in the cluster is started normally.
+
+3. Deploy the second TiDB Operator.
+
+    Refer to [Deploy TiDB Operator](deploy-tidb-operator.md) to deploy the second TiDB Operator without `tidb-scheduler`. Add the following configuration in the `values.yaml`:
+
+    ```yaml
+    controllerManager:
+      selector:
+      - user=qa
+    appendReleaseSuffix: true
+    scheduler:
+      create: false
+    ```
+
+    > **Note:**
+    >
+    > If you configure `scheduler.create: true`, a `tidb-scheduler` named `{{ .scheduler.schedulerName }}-{{.Release.Name}}` is created. To use this `tidb-scheduler`, you need to configure `spec.schedulerName` in the TidbCluster CR to the name of this scheduler.
+
+4. Deploy the TiDB cluster.
+
+    1. Refer to [Configure the TiDB Cluster](configure-a-tidb-cluster.md) to configure the TidbCluster CR, and configure `labels` to match the `selector` set in the last step. For example:
+
+        ```yaml
+        apiVersion: pingcap.com/v1alpha1
+        kind: TidbCluster
+        metadata:
+          name: basic2
+          labels:
+            user: qa
+        spec:
+          ...
+        ```
+
+        If `label` is not set when you deploy the TiDB cluster, you can configure `label` by running the following command:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        kubectl -n ${namespace} label tidbcluster ${cluster_name} user=qa
+        ```
+
+    2. Refer to [Deploy TiDB in General Kubernetes](deploy-on-general-kubernetes.md) to deploy the TiDB cluster. Confirm that each component in the cluster is started normally.
+
+5. View the logs of the two TiDB Operator, and confirm that each TiDB Operator manages the TiDB cluster that matches the corresponding selectors.
+
+    For example:
+
+    View the log of `tidb-controller-manager` of the first TiDB Operator:
+
+    ```shell
+    kubectl -n tidb-admin logs tidb-controller-manager-55b887bdc9-lzdwv
+    ```
+
+    <details>
```

Comment:
Using the `<details>` syntax for toggle display might affect searching the toggled words, as you have previously found out.

Reply:
Yes. I agree with you. The Chinese author chose the toggle display. I assume he believed the output is trivial in this context?

Reply:
Yes, it's OK.